packages/docusaurus-theme-classic/src/theme/BlogPostItem/index.tsx

```diff
@@ -58,18 +58,18 @@ function BlogPostItem(props: Props): JSX.Element {
     readingTime,
     title,
     editUrl,
+    author,
   } = metadata;
-  const {author} = frontMatter;
 
   const image = frontMatterAssets.image ?? frontMatter.image;
 
-  const authorURL = frontMatter.author_url || frontMatter.authorURL;
-  const authorTitle = frontMatter.author_title || frontMatter.authorTitle;
+  const authorName = author.name;
+  const authorURL = author.url;
+  const authorTitle = author.title;
   const authorImageURL =
     frontMatterAssets.author_image_url ||
     frontMatterAssets.authorImageURL ||
```

Comment:
this part may not be simple to handle for multiple authors 🤪

Reply:
I did it 🤪